requirements.txt

```diff
@@ -1,9 +1,14 @@
-attrdict
-paramiko
+# The below packages are required to run this project
+# Other required packages could be found in the Dockerfile, such as https://gitlab.cee.redhat.com/virtwho/virtwho-jobs/-/blob/master/docker/fedora/Dockerfile-fedora39
+# Line beginning with 'RUN yum -y install '
```

Comment:
it seems all required packages for Python are listed here, we could remove the comments

Reply:
But as my understand, the packages in the `yum install` line are also needed, and some of the ones could also be install by `pip`, such as the `iputils findutils jq zip`, so I added the comments here.